lms/djangoapps/support/static/support/jsx/entitlements/components/Table/Table.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { Button, Hyperlink, Table } from '@edx/paragon';
+
+const entitlementColumns = [
+  {
+    label: 'User',
+    key: 'user',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Course uuid',
+    key: 'course_uuid',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Enrollment',
+    key: 'enrollment_course_run',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Mode',
+    key: 'mode',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Expired At',
+    key: 'expired_at',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Created',
+    key: 'created',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Modified',
+    key: 'modified',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Order',
+    key: 'order_number',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Actions',
+    key: 'button',
+    columnSortable: false,
+    hideHeader: false,
+    onSort: () => {},
+  },
+];
+
+const EntitlementTable = (props) => {
+  const entitlementData = props.entitlements.map((entitlement) => 
+    Object.assign({}, entitlement, {
+      expired_at: entitlement.expired_at ? moment(entitlement.expired_at).format('MMM. D, YYYY') : '',
+      created: moment(entitlement.created).format('MMM. D, YYYY'),
+      modified: moment(entitlement.modified).format('MMM. D, YYYY'),
+      order_number: <Hyperlink
+        destination={props.ecommerceUrl + entitlement.order_number + '/'}
+        content={entitlement.order_number || ''}
+      />,
+      button: <div> No Actions Currently Available </div>
+    })
+  );
+  return (
+    <div>
+      <Table
+        data={entitlementData}
+        columns={entitlementColumns}
+      />
+    </div>
+  )
+}
+
+EntitlementTable.propTypes = {
+  entitlements: PropTypes.array.isRequired,
+  ecommerceUrl: PropTypes.string.isRequired,
+};
+
+export default EntitlementTable;
```

Comment:
I get why you named this `EntitlementTable` but [`airbnb`'s JavaScript style guide recommends that the default export exactly match the filename](https://github.com/airbnb/javascript#naming--filename-matches-export).

```javascript
// file 1 contents
class CheckBox {
  // ...
}
export default CheckBox;

// file 2 contents
export default function fortyTwo() { return 42; }

// file 3 contents
export default function insideDirectory() {}

// in some other file
// bad
import CheckBox from './checkBox'; // PascalCase import/export, camelCase filename
import FortyTwo from './FortyTwo'; // PascalCase import/filename, camelCase export
import InsideDirectory from './InsideDirectory'; // PascalCase import/filename, camelCase export

// bad
import CheckBox from './check_box'; // PascalCase import/export, snake_case filename
import forty_two from './forty_two'; // snake_case import/filename, camelCase export
import inside_directory from './inside_directory'; // snake_case import, camelCase export
import index from './inside_directory/index'; // requiring the index file explicitly
import insideDirectory from './insideDirectory/index'; // requiring the index file explicitly

// good
import CheckBox from './CheckBox'; // PascalCase export/import/filename
import fortyTwo from './fortyTwo'; // camelCase export/import/filename
import insideDirectory from './insideDirectory'; // camelCase export/import/directory name/implicit "index"
// ^ supports both insideDirectory.js and insideDirectory/index.js
```

Reply:
makes sense, in that case, do you think it makes more sense to continue using EntitlementTable and rename the file?